src/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -1198,6 +1198,10 @@ public bool Start()
             {
                 throw new InvalidOperationException(SR.StandardErrorEncodingNotAllowed);
             }
+            if (!string.IsNullOrEmpty(startInfo.Arguments) && startInfo.ArgumentList.Count > 0)
```

Comment:
Why treat empty the same as null? An empty arguments string has a valid meaning.

Reply:
Even if the arguments passed is null during constructor of processInfo is null, we return empty as value in getter.